testsuite/tests/libratest/smoke_test.rs

```diff
@@ -699,7 +699,7 @@ fn test_full_node_basic_flow() {
 
 #[test]
 fn test_e2e_reconfiguration() {
-    let (mut env, mut client_proxy_1) = setup_swarm_and_client_proxy(5, 1);
+    let (mut env, mut client_proxy_1) = setup_swarm_and_client_proxy(3, 1);
```

Comment:
Intended?

Reply:
Yeah, the tests would fail with 3 validators but flaky with 5 in previous broken changes. Because 3 requires everyone to make progress.